### DISCUSSION TITLE
feature request: API resource generation

### DISCUSSION THREAD

**Discussion by notramo** [luckyframework/lucky#1780] [Feature requests & Ideas]
Lucky has `gen.resource.browser`, but not `gen.resource.api`.

**Comment by jwoertink** [comment]
You suggested this before: https://github.com/luckyframework/lucky/discussions/1778 :joy: I don't currently have time to add this in. I'm mainly spending time on upkeep, and trying to get the next version out. If you're interested in making a PR, it's defined here https://github.com/luckyframework/avram/tree/main/src/lucky/tasks/gen/resource
